content/news/2023-02-20-global-team-recruiting/index.en.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Global Team Recruiting"
+author: "Global Leadership Team"
+date: "2023-02-20"
+description: ""
+tags: 
+- global
+- team
+categories:
+- r-ladies
+---
+
+# TL; DR
+
+The R-Ladies Global Team is recruiting new members!
+Please fill [this form](https://docs.google.com/forms/d/1OZB6eVrsocvRyqD-ec5ui69SfXuOR8AZ_L-33B2d5yM/edit?pli=1) to submit yourself for consideration.
+
+# Background
+
+The R-Ladies Global Team is made of volunteers that dedicate a few hours per week to the management of the global organization.
+We think of ourselves as one larger Global Team comprised of many smaller teams that address specific organizational needs.
+We want to build a diverse Global Team with good representation of all community members.
+
+# Timeline (approximate)
+
+-   date1-date2: Google form is open to submissions (2 weeks)
+
+-   date3-date4: Global team reviews submissions (2 weeks)
+
+-   date5-date6: Target date to notify individuals of selection.
+    (1 week)
+
+-   date7-date8: Invitation to meet with 1-3 global team members to clarify any questions or expectations to ensure a good match for all parties.
+    (2 weeks)
+
+# Selection considerations
+
+We will consider various aspects with the aim of increasing the diversity of the team.
+The following are not an exhaustive list of requirements but rather an illustration of the various paths possible.
+A new team member is not expected to satisfy all criteria.
+
+-   Geographical location
+
+-   Demographic characteristics like ethnicity, age, and gender identity
+
+-   Background and experience such as education, skills, and knowledge \<--Note: if we keep this here then should google form should be ammended to capture this??--\>
+
+-   R-Ladies/R community involvement
+
+# Benefits
+
+By participating in R-Ladies you are helping other women or gender minorities by creating a safe space to learn, teach, and develop, as well as build out a support network and contacts.
+
+Some of the personal benefits that we think come with joining the global team and volunteering time are gaining experience in leadership, asynchronous remote work, international team work, not-for-profit open-source work, and work in a women(+) only team and organization.
+
+Depending on the role, there also may be opportunities for technical upskilling.
+For example, some members of the Global team use R, reproducible reports, git/GitHub, GitHub Actions, Hugo website development, and database integrations.
+(Knowledge of these tools is not a requirement for selection; the Global team will provide training on processes specific to your role.)
+
+# Commitment
+
+We ask those seeking consideration to commit to one year of 1 to 4 hours a week for service to the Global Team.
+Of course, vacations and exceptions may occur.
```

Comment:
```suggestion
Since R-Ladies is a purely volunteer run organisation, we respect and encourage your need to have vacations, free-time and focus on other aspects of your life.
The expected hours spent on R-Ladies work should happen when you are yourself available to do them. 
```